config/env.example

```diff
@@ -86,6 +86,9 @@ MAILJET_SECRET_KEY=""
 # API Entreprise https://api.gouv.fr/api/api-entreprise.html
 API_ENTREPRISE_KEY=""
 
+# API Particulier https://api.gouv.fr/les-api/api-particulier
+API_PARTICULIER_KEY=""
```

Comment:
Cette variable d'environnement déclaré dans le commit "Feat (API Particulier): add a lib to query the API", semble ne plus être utilisée à partir du commit "Feat (API Particulier): gestion fine des sources de données"

Reply:
Oui tout à fait ! Tu m'as pris de court, j'étais justement en train de la supprimer ;)

D'ailleurs, je me suis aperçu que toute variable d'environnement nécessaire pour la bonne exécution des tests doit obligatoirement se trouver dans ce fichier `config/env.exemple`, car c'est lui qui est utilisé pour déclarer les variables d'environnement de test pour le CI de GitHub.